packages/pds/src/notifications.ts

```diff
@@ -0,0 +1,283 @@
+import axios from 'axios'
```

Comment:
I think we can totally remove the notification server from the pds - only needs to run in the appview

Reply:
Okay. I can remove it completely.

Reply:
Done.